db/schema.rb

```diff
@@ -25,14 +28,14 @@
     t.index ["versionable_id"], name: "index_annotations_on_versionable_id"
   end
 
-  create_table "answers", id: :integer, force: :cascade do |t|
-    t.text "text", limit: 16777215
+  create_table "answers", id: :serial, force: :cascade do |t|
+    t.text "text"
     t.integer "plan_id"
     t.integer "user_id"
     t.integer "question_id"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.integer "lock_version", default: 0
+    t.integer "lock_version", default: 0, null: false
```

Comment:
The `null: false` change is the only one I was explicitly targeting with this migration. This migration almost seems to revert all of the migration changes from Release 3.1.0: https://github.com/portagenetwork/roadmap/pull/271/files#diff-cbf8b25d6d853acf58fa9057841f407d29ffe90649c75cf9e3f51b2d6e3aa1d3L13